src/core/aws-ses.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@nestjs/common';
+import * as AWS from 'aws-sdk';
+import { EnvironmentService } from './config/environment.service';
+
+@Injectable()
+export class SesService {
+  private readonly ses: any;
+  constructor(private readonly env: EnvironmentService) {
+    AWS.config.update({
+      accessKeyId: this.env.string('AWS_ACCESS_KEY').optional('cord-field'),
+      secretAccessKey: this.env.string('AWS_SECRET_KEY').optional('cord-field'),
+      region: this.env.string('AWS_REGION').optional('cord-field'),
+    });
+    this.ses = new AWS.SES({ correctClockSkew: true });
+  }
+
+  async sendEmail(params: any): Promise<boolean> {
+    return this.ses.sendEmail(params, (err: any, data: any) => {
+      if (err) {
+        console.log(err, err.stack);
+        return false;
+      } else {
+        console.log(data);
+        return true;
+      }
+    });
+  }
```

Comment:
Most AWS methods support both callbacks and promises to maintain backwards compatibility. We want to only use the promises. Also FYI the returned boolean in the callback will not make the outer `sendEmail` method return a boolean.

Instead do:
```ts
await ses.sendEmail(params).promise();
```
I don't think a wrapping method is needed either and can just be called with the directly injected SES.

Side note: We may in the future decide we want an `EmailService` with specific methods for each type of email sent.